src/client/Microsoft.Identity.Client/Http/HttpManager.cs

```diff
@@ -167,20 +88,45 @@ private static HttpRequestMessage CreateRequestMessage(Uri endpoint, IDictionary
                 logger.Info(() => string.Format(CultureInfo.InvariantCulture,
                     MsalErrorMessage.HttpRequestUnsuccessful,
                     (int)response.StatusCode, response.StatusCode));
+
+                isRetriable = _retryCondition(response);
             }
             catch (TaskCanceledException exception)
             {
                 if (cancellationToken.IsCancellationRequested)
                 {
-                    logger.Info("The HTTP request was cancelled. ");
+                    logger.Info("The HTTP request was canceled. ");
                     throw;
                 }
 
                 logger.Error("The HTTP request failed. " + exception.Message);
+                isRetriable = true;
```

Comment:
Why retry on task cancelled exception? Isn't this cancellation from the user?

Reply:
In the line above we check if the cancellation token is cancelled and we throw. If the cancellation token is not canceled, we retry. This has always been like this.